homeassistant/helpers/script.py

```diff
@@ -220,287 +236,349 @@ def referenced_entities(self):
         self._referenced_entities = referenced
         return referenced
 
-    def run(self, variables=None, context=None):
+    def run(self, variables=None, context=None, logger=None, message_base=None):
```

Comment:
I don't even think that this one is used. I think that we can drop this method.

Reply:
It's actually used in `call_from_config` towards the beginning of the module. I searched for calls to `call_from_config` and it's only used in tests (which can be done a different way.) So I should probably get rid of `call_from_config` in addition to `run`.

Reply:
Yeah 👍